### PR TITLE
Issue rsdmx 4

### PR DESCRIPTION
**Pull request by OSoerensen**
Added rjsdmx from issue #4

### CODE REVIEW

**Review by OSoerensen — COMMENTED** [review]
No changes needed

File: catalogue.json [hunk 1]
```diff
@@ -114,6 +114,15 @@
                 "license": "Free",
                 "url": "https://cros.ec.europa.eu/dashboard/sdmx-converter",
                 "image": "https://ec.europa.eu/eurostat/o/adaptive-media/image/17995989/Thumbnail-128x128/EN_04_SDMX_Converter_API.png?t=1701270530811"
+            },
+            {
+                "name": "RJSDMX",
```
Comment (Buffett65) [inline catalogue.json]: Typo? shouldn't this be rsdmx?

**Comment by OSoerensen** [comment]
@Buffett65  thank you for spotting this, it should have been corrected now

File: catalogue.json (hunk 1 above)
Comment (eblondel) [reply catalogue.json]: yes it's rsdmx, native R package, not the RJSDMX

File: catalogue.json [hunk 2]
```diff
@@ -118,7 +118,7 @@
             {
                 "name": "RJSDMX",
                 "description": "rsdmx is a package to parse/read SDMX data and metadata in R. It provides a) a set of classes and methods to read data and metadata documents exchanged through the Statistical Data and Metadata Exchange (SDMX) framework. The package currently focuses on the SDMX XML standard format (SDMX-ML); and b) an interface to SDMX web-services for a list of well-known data providers, such as EUROSTAT, OECD, and others.",
-                "supported_features" : "This is an extended text to add more information about the tool, it could contain information about platforms etc.",
+                "supported_features" : "See link to the tool",
                 "owner": "Emanuel Blondel",
```
Comment (eblondel) [inline catalogue.json]: A small typo on my name :-) maybe my fault when filling the form: Emmanuel Blondel

**Comment by OSoerensen** [comment]
@glenntav will you review this one

**Pull request closed without merging**